frontend/src/metabase/nav/containers/MainNavbar/MainNavbar.unit.spec.tsx

```diff
@@ -106,13 +107,26 @@ async function setup({
     setupCardsEndpoints([openQuestionCard]);
   }
 
-  const dashboards = openDashboard ? { [openDashboard.id]: openDashboard } : {};
```

Comment:
This was causing a type error now that `DashboardState` accurately reflects the normalized redux store.